src/Mentat.Repository/Models/CardDatabaseSettings.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class CardDatabaseSettings : ICardDatabaseSettings
     {
-        public string ConnectionString { get; set; } = "mongodb+srv://cdb:eN9qAv7forAKSnEt@flashcardapp.4yggo3z.mongodb.net/?retryWrites=true&w=majority";
+        public string ConnectionString { get; set; } = "mongodb+srv://feature-tags-user:cFVOCEplar0AgjOq@cluster0.5mowjp1.mongodb.net/test";
```

Comment:
This is going to be a problem. Not only do we have existing functionality that depends on this value, `CardDatabaseSettings` will be handled differently once `Identity` branch merges. I strongly suspect that once that branch and the Bash script generator branch merge, this branch will have one or more merge conflicts anyway. That's not an unusual situation. We will try to push through those other two PRs so that perhaps we can get these conflicts resolved before end of semester. If we don't, I'll just have the next class resolve any blockers for this PR.